contrib/centos/aziot-identity-service.spec

```diff
@@ -0,0 +1,146 @@
+# spec file for package aziot-identity-service
+#
+# Copyright (c) Microsoft. All rights reserved.
+
+Name: aziot-identity-service
+Version: @version@
+Release: @release@
+Summary: Azure IoT Identity Service and related services
+License: Proprietary
```

Comment:
Is this correct? The repo (and the license file below) are MIT.

Reply:
Yeah, I copied this from iotedge and was confused by this too, because the license in `debian/copyright` is still MIT. It might be something to do with the source / product separation that we have with iotedge vs azure-iotedge, but then why doesn't it apply to the Debian package...

I'll follow up with Mike.

Reply:
I didn't hear from Mike and Dave doesn't remember, so I've made this MIT to match the Debian package.